authority/src/tests.rs

```diff
@@ -80,13 +80,13 @@ fn schedule_dispatch_at_work() {
 				delay: 1,
 				origin: Box::new(OriginCaller::system(RawOrigin::Root)),
 			}),
-			index: 1,
+			index: 0,
```

Comment:
Could you please give some details why the index is changed?

Reply:
Similarly for the other test aswell.

Reply:
https://github.com/open-web3-stack/open-runtime-module-library/blob/27115ac7745662c2134862fdd8fc77edf6c9ad04/authority/src/lib.rs#L259-L273

There is no `[transactional]` here, the failed tests are not rolled back